pkg/controller/telemetry/metric.go

```diff
@@ -508,7 +506,7 @@ func (m *MetricController) Run(ctx context.Context, mapOfTcpInfo *ebpf.Map) {
 			}
 
 			connectionLabels := connectionMetricLabels{}
-			if m.EnableConnectionMetric.Load() && data.duration > LONG_CONN_METRIC_THRESHOLD {
+			if m.EnableConnectionMetric.Load() {
```

Comment:
Would you refactor a little more to make it together with L522. We have so many duplicate flag checking in this function